package/kernel/cryptodev-linux/Makefile

```diff
@@ -10,19 +10,18 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=cryptodev-linux
-PKG_VERSION:=1.9.git-2018-11-02
+PKG_VERSION:=1.10
 PKG_RELEASE:=1
 
-PKG_SOURCE_URL:=https://github.com/cryptodev-linux/cryptodev-linux
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=f1a693000d116718379f8b53ed7bc6b9c0f7de27
-PKG_MIRROR_HASH:=340d314e2a88bf2449ccee906b141e085d376f6e2a94a64e36254e8376323169
+PKG_SOURCE_URL:=https://codeload.github.com/$(PKG_NAME)/$(PKG_NAME)/tar.gz/$(PKG_NAME)-$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=833ab7c5c88d2b700a7c702a151254c089a3058886a63cc7d12630e364b8ea83
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=COPYING
 
 PKG_MAINTAINER:=Ansuel Smith <ansuelsmth@gmail.com>
 
-PKG_BUILD_DIR:=$(KERNEL_BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+ PKG_BUILD_DIR:=$(KERNEL_BUILD_DIR)/$(PKG_NAME)-$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
Why twice $PKG_NAME in $PKG_BUILD_DIR ?

Reply:
Because GitHub adds the "username" which happens to the be same as repo name in this case

Reply:
@diizzyy BTW, you've introduced one space indent before `PKG_BUILD_DIR`

Reply:
Fixed, thanks for point that out :-)